src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
Is there any danger that something in this script will mess with a real configuration someone already has on their machine?  If so, before overwriting / deleting / changing / etc., we should prompt to make sure the caller is ok with the changes.